wdtk-client/src/main/java/org/wikidata/wdtk/client/DumpProcessingAction.java

```diff
@@ -101,4 +101,26 @@
 	 */
 	void setDumpInformation(String project, String dateStamp);
 
+	/**
+	 * Returns a report message containing information about the files which
+	 * were generated.
+	 * 
+	 * @return report message
+	 */
+	String getReport();
+
+	/**
+	 * Sets the actions name.
```

Comment:
Would need to be "the action's name" or "the name of the action". I would also add "If this is not set, a default name will be used."